.travis.yml

```diff
@@ -48,6 +52,12 @@ jobs:
 
     - name: "Unit Tests"
       before_script:
+        - wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+        - sudo apt install ./google-chrome-stable_current_amd64.deb
```

Comment:
i wonder if we should try pinning these versions. in general, I don't like just using 'latest', because then they can change the version and your tests break and you can't tell whether it was the version change or the commit.